01-introduction.Rmd

```diff
@@ -317,23 +313,23 @@ The initial release supported only raster drivers but subsequent enhancements pr
 Many of these additional capabilities were developed by Barry Rowlingson and released in the **rgdal** codebase in 2006 [see @rowlingson_rasp:_2003 and the [R-help](https://stat.ethz.ch/pipermail/r-help/2003-January/028413.html) email list for context].
 
 **sp**, released in 2005, overcame R's inability to distinguish spatial and non-spatial objects [@pebesma_classes_2005].
-**sp** grew from a [workshop](http://spatial.nhh.no/meetings/vienna/index.html) in Vienna in 2003 and was hosted at sourceforge before migrating to [R-Forge](https://r-forge.r-project.org).
+**sp** grew from a workshop in Vienna in 2003 and was hosted at SourceForge before migrating to [R-Forge](https://r-forge.r-project.org).
```

Comment:
I think where it's hosted is of limited use/interest.

Reply:
We can remove the link to the legacy r-forge website and shorten this bit I think.